### PR TITLE
[Lance] Upgrade Lance dependency to 8.0.0

### PR DESCRIPTION
**Pull request by czy006**
## Why are the changes needed?

Upgrade Lance dependency to 8.0.0

## Brief change log

- Pom Upgrade Lance dependency to 8.0.0

## How was this patch tested?

- [x] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] Run test locally before making a pull request

## Documentation

- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

File: amoro-format-lance/pom.xml [hunk 1]
```diff
@@ -47,29 +47,19 @@
...
-                <exclusion>
-                    <groupId>org.lance</groupId>
-                    <artifactId>lance-namespace-core</artifactId>
-                </exclusion>
-                <exclusion>
-                    <groupId>org.lance</groupId>
-                    <artifactId>lance-namespace-apache-client</artifactId>
-                </exclusion>
-            </exclusions>
+            <version>8.0.0</version>
```
Comment (zhoujinsong) [inline amoro-format-lance/pom.xml:50]: How about adding a version property?

**Review by zhoujinsong — COMMENTED** [review]
@czy006 I left a comment and the ci seems fail.
